lib/server.js

```diff
@@ -21,6 +23,33 @@ module.exports = async function({ plants, nurseries }) {
   }
   app.use(express.static(publicDir));
   app.use('/images', express.static(`${__dirname}/../images`));
+  app.use(express.json())
+
+  app.post('/get-vendors', async(req, res) =>{
+    const baseUrl = process.env.PAC_API_BASE_URL
+    const apiKey = process.env.PAC_API_KEY
+    const url = `${baseUrl}/Plant/FindVendorsForPlantName?plantName=${req.body.plantName}&zipCode=${req.body.zipCode}&radius=${req.body.radius}&limit=5`;
+    var response = await axios.get(url, {headers: {'Authorization': 'Bearer ' + apiKey}})
+    return res.send(response.data)
+  })
+  app.post('/get-city', async(req, res)=> {
+    //console.log(req.params)
+    //call vendor/FindZip
+    const baseUrl = process.env.PAC_API_BASE_URL
+    const apiKey = process.env.PAC_API_KEY
+    const url = `${baseUrl}/vendor/FindCity?zipCode=${req.body.zipCode}`;
+    var response = await axios.get(url, {headers: {'Authorization': 'Bearer ' + apiKey}})
+    return res.send(response.data)
+  })
+  app.post('/get-zip', async(req, res)=> {
+    //console.log(req.params)
+    //call vendor/FindZip
+    const baseUrl = process.env.PAC_API_BASE_URL
+    const apiKey = process.env.PAC_API_KEY
+    const url = `${baseUrl}/vendor/FindZip?lat=${req.body.latitude}&lng=${req.body.longitude}`;
```

Comment:
Always validate inputs. `req.body.latitude` could be anything here. Make sure it is a string, and build the query string using the `qs` module or built-in capabilities of Axios for assembling a query string without the risk of the site being manipulated in unwanted ways by sneaky inputs. This stackoverflow thread is good on how to do it easily with Axios, you'll have less work to do in the end:

https://stackoverflow.com/questions/53709142/best-way-to-pass-query-parameters-to-url-using-axios-in-vue